src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilderRow.tsx

```diff
@@ -39,61 +46,95 @@ const QueryBuilderRow: React.FC<IQueryBuilderRowProps> = (props) => {
         onChangeValue,
         position,
         removeRow,
-        styles
+        styles,
+        theme,
+        updateSnippet
     } = props;
+    const propertySelectorId = useId('cb-advanced-search-property-select');
     const classNames = getClassNames(styles, {
-        theme: useTheme()
+        theme: useTheme(),
+        isOnlyFirstRow: isRemoveDisabled
     });
+    const { t } = useTranslation();
 
-    const [comboBoxOptions, setComboBoxOptions] = useState<IComboBoxOption[]>(
-        []
+    /** React select values */
+    const { isLoading, flattenedModelProperties } = useFlattenedModelProperties(
+        {
+            adapter,
+            allowedPropertyValueTypes
+        }
     );
+
+    const [propertyOptions, setPropertyOptions] = useState([]);
```

Comment:
give this a type please, otherwise it's considered an `any[]` which isn't very safe.